src/uu/printf/src/printf.rs

```diff
@@ -284,7 +286,12 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         None => vec![],
     };
 
-    memo::Memo::run_all(format_string, &values[..]);
+    let result = memo::Memo::run_all(format_string, &values[..]);
+    if result.is_err() {
+        show_error!("%{}: invalid conversion specification", result.unwrap_err());
+        exit(1);
+    };
```

Comment:
You should just be able to return the result from `run_all` and it will be printed automatically (this is done by the `uucore::main` macro)

Reply:
I see, that's something I didn't consider. We should indeed keep the error message the same. I think the `UError` itself should contain the `"invalid conversion specification"` message and then we can just propagate it. And then you can change the uses in `seq` to:
```rust
if let Err(x) = Memo::run_all(f, &[s]) {
    show!(x);
    exit(1);
}
```
Does that make sense?